lib/nuget-parser/index.ts

```diff
@@ -58,13 +63,25 @@ export async function buildDepTreeFromFiles(
     version: '0.0.0',
   };
 
-  let targetFramework;
+  let targetFramework: TargetFramework | undefined;
   try {
     if (manifestType === 'dotnet-core') {
       targetFramework = await getTargetFrameworksFromProjFile(projectRootFolder);
     } else {
       // .csproj is in the same directory as packages.config or project.json
       targetFramework = await getTargetFrameworksFromProjFile(path.resolve(fileContentPath, '../'));
+
+      // finally, for the .NETFramework project, try to assume the framework using dotnet-deps-parser
+      if (!targetFramework) {
+        const extractedFrameworks = await depsParser.extractTargetFrameworksFromFiles(safeRoot, safeTargetFile);
+    
+        if (extractedFrameworks && extractedFrameworks.length > 0) {
+          // find the lowest framework in the list
+          // (is this a better approach, considering SCM test picks the _first_ framework in the list?)
```

Comment:
This comment is not accurate, we pick first one in CLI. on SCM we process each TF as a project correctly.

Keep picking the first one I guess :| Just for constitency so that when we are on call we know it is first one. You can't compare them like this anyway, there is an entire microservice to do this because it is so complex

Reply:
Sure, sticking to the first framework; fixed